src/poprox_recommender/data/mind.py

```diff
@@ -123,9 +123,8 @@ def lookup_article(self, *, id: str | None = None, uuid: UUID | None = None):
         article = Article(
             article_id=uuid,
             url=f"urn:uuid:{uuid}",
-            title=str(self.news_df.loc[id, "title"]),
+            headline=str(self.news_df.loc[id, "title"]),
         )
-        article.content = article.title
```

Comment:
If I updated this line it would read:
```python
article.subhead = article.headline
```
which doesn't seem right to me. If there isn't a subhead on these articles I don't think that's a problem for either the recommender or the evaluation. (Maybe I'm wrong though?)

Reply:
According to: https://github.com/msnews/msnews.github.io/blob/master/assets/doc/introduction.md#newstsv
There should be a field `Abstract`
But I don't know much about MIND or have a good way to check this.

@sophiasun0515?